src/internet_identity/src/storage.rs

```diff
@@ -12,6 +12,11 @@ const HEADER_SIZE: u32 = 512;
 const DEFAULT_ENTRY_SIZE: u16 = 2048;
 const EMPTY_SALT: [u8; 32] = [0; 32];
 const WASM_PAGE_SIZE: u32 = 65536;
+const STABLE_MEMORY_SIZE: u64 = 1 << 32;
+
+/// The maximum number of users this canister can store.
+pub const DEFAULT_RANGE_SIZE: u64 =
+    (STABLE_MEMORY_SIZE - HEADER_SIZE as u64) / DEFAULT_ENTRY_SIZE as u64;
```

Comment:
Shall we reserve some space in the stable memory for posteriorty? 10%? Maybe we need to add features that need additional space later.

Reply:
I like that idea, done.